gdnative-core/src/core_types/vector2.rs

```diff
@@ -19,7 +19,8 @@ pub trait Vector2Godot {
     fn snapped(self, by: Self) -> Self;
     /// Returns a perpendicular vector.
     fn tangent(self) -> Self;
-
+    /// Moves the vector toward to by the fixed delta amount.
```

Comment:
The doc-comment can be reworded to be more in place with other ones. Also, "to" and "delta" should be in backticks:

```rust
/// Returns `self` moved towards `to` by the distance `delta`, clamped by `to`.
```

Reply:
changed